modules/iam-assumable-role-with-oidc/main.tf

```diff
@@ -1,6 +1,9 @@
 locals {
   aws_account_id = var.aws_account_id != "" ? var.aws_account_id : data.aws_caller_identity.current.account_id
-  provider_url   = replace(var.provider_url, "https://", "")
+  identifiers = [
+    for url in distinct(concat(var.provider_urls, var.provider_url)) :
+    "arn:aws:iam::${local.aws_account_id}:oidc-provider/${replace(url, "https://", "")}"
```

Comment:
Make sure to use `${data.aws_partition.current.partition}` instead of `aws` as a partition name.

Reply:
Ah good catch. Fixed.